tests/chisel-wrapper/runtest

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Needs chisel and syft.
+
+set -eux
+
+STATUS_FILE="$(mktemp)"
+ROOTFS="$(mktemp -d)"
+
+install() {
+    ./chisel-wrapper --generate-dpkg-status "$STATUS_FILE" -- \
+        --release ubuntu-20.04 --root "$ROOTFS" \
```

Comment:
we want to test for only one version of ubuntu?

Reply:
Hey Samir! Yeah, I think that's fine.